### ISSUE TITLE
Kite is not showing up in JupyterNotebook

### ISSUE THREAD

**Issue by Collin-Li-DataScience** [kiteco/issue-tracker#500]
**Required Information**
To help us triage your issue effectively, please make sure you've completed these steps before opening this issue. Issues with not enough information may be deprioritized or ignored.

- [x] [Read our guide](https://help.kite.com/article/96-contact-us-with-logs) on how to report an issue.

- Your OS: macOS Catalina Version 10.15.7
- Your browser console logs: 

<img width="1520" alt="Screen Shot 2020-12-03 at 9 47 05 PM" src="https://user-images.githubusercontent.com/39756954/101126821-199af000-35b1-11eb-80b2-ebd4fd6d5312.png">

- Your terminal console output (as much of as possible, not just the first or final line)
   
[terminal output.txt](https://github.com/kiteco/issue-tracker/files/5640930/terminal.output.txt)

- The output of jupyter labextension list and jupyter serverextension list
- The output of pip list
All the three commands output are in the file attached below
[troubleshoot_output.txt](https://github.com/kiteco/issue-tracker/files/5640815/troubleshoot_output.txt)

- If you use conda, please share your environment

[environment.txt](https://github.com/kiteco/issue-tracker/files/5640923/environment.txt)

- [ ] Include your Kite application logs, _taken while you are experiencing your issue_.
**https://s3.console.aws.amazon.com/s3/object/kite-client-logs/prod/584122/kite_local/client.log.2020-12-03_09-37-
ude your resource usage logs, _taken while you are experiencing your issue_.
- [ ] If reporting an issue with Kite's editor plugin, include which editor you are using and the version, and any accompanying logs from your editor's console.

**Describe the bug**
A clear description of what the bug is. Include steps to reproduce the bug if necessary.
Kite desktop has been downloaded, kite and Jupiterlab is running at the same user on the computer
JupterLab is not reporting that it is missing Kite desktop application
Everything is enabled
However, Kite is not showing up in JupiterLab, from the beginning when I start JupyterLab with kite assistant for the first time
I also checked I only have one kernel by running jupyter kernelspec list, which is Python3
**Screenshots**
<img width="432" alt="Screen Shot 2020-12-03 at 9 38 11 PM" src="https://user-images.githubusercontent.com/39756954/101126278-fe7bb080-35af-11eb-9b2f-7376926e83cb.png">
As shown above, Kite is not showing up in the JupterLab

**Comment by floriandotorg** [comment]
Have the same problem since updating to jupyter 3.0.0rc13. Browser console shows error:
```
index.es6.js:282 Plugin '@kiteco/jupyterlab-kite:plugin' failed to activate.
(anonymous) @ index.es6.js:282
index.es6.js:283 TypeError: Cannot read property 'composite' of undefined
    at new _ (model.js:72)
    at new Wn (kite_accessible.js:23)
    at Function.Wn.CreateAsync (kite_accessible.js:142)
    at async activate (index.js:52)
    at async Promise.all (index 101)
(anonymous) @ index.es6.js:283
```

**Comment by xellosiris** [comment]
same problem here

**Comment by edzkite** [comment]
@Collin-Li-DataScience It looks like you're running a Jupyter Notebook. Unfortunately, Kite is only supported for JupyterLab.

**Comment by floriandotorg** [comment]
@edzkite not true, currently Kite is *not* working in Jupyter *Lab* 3+. See error above.

**Comment by edzkite** [comment]
@floriandotorg Could you please create a separate issue, along with the same information that the original author provided? I'm not sure if these are necessarily related, since he's running JupyterLab 2.2.9.

**Comment by edzkite** [comment]
@Collin-Li-DataScience

- Looking at your terminal output, you run `/Users/mucongl/anaconda3/bin/jupyter_mac.command ; exit;` which I believe launches a Notebook instance. This also corroborates that:
```
[I 22:22:24.682 NotebookApp] Serving notebooks from local directory: /Users/mucongl
[I 22:22:24.682 NotebookApp] Jupyter Notebook 6.1.4 is running at:
```
As mentioned earlier, Jupyter Notebooks are not supported by Kite at this time.

- I also see in your environment that you have:
```
JupyterLab v2.2.9
Known labextensions:
   app dir: /Users/mucongl/anaconda3/share/jupyter/lab
        @kiteco/jupyterlab-kite v1.2.0  enabled  OK

Uninstalled core extensions:
    @kiteco/jupyterlab-kite
```
Some users do report issues where there are multiple instances or lingering artifacts of an extension present. See https://github.com/kiteco/issue-tracker/issues/506#issuecomment-744635292 for a potential resolution.

**Comment by Collin-Li-DataScience** [comment]
@edzkite i was running on JupiterLab. I attached the wrong file for terminal report,please ignore that.

**Comment by edzkite** [comment]
That's fine. It seems like the crux of the issue is that `@kiteco/jupyterlab-kite` is reported as both enabled and uninstalled. https://discourse.jupyter.org/t/extension-shows-up-as-both-installed-and-uninstalled-and-doesnt-work/2688 seems to suggest that when it's in this state, the extension will act as if it's not installed at all, hence the lack of console output.

Could you please try the suggestion in the thread (removing `build_config.json`)?
